### DISCUSSION TITLE
个人微信模式用wechaty语音一直出错

### DISCUSSION THREAD

**Discussion by hd19820806** [zhayujie/chatgpt-on-wechat#1669] [Q&A]
各位大佬，我用itchat模式加百度语音+fastgpt的key 一切正常。语音识别和语音合成都没问题。换了wechaty模式以后，百度微软的语音全都不好用了。文字模式正常。换了官方openai的key以后openai语音正常。
求助大佬是什么原因

**Comment by 7tinghua** [comment]
请详细说一下你切换wechaty模式的步骤细节，否则无法根据你的描述来确认问题。
